src/typeahead/docs/readme.md

```diff
@@ -27,6 +27,10 @@ The typeahead directives provide several attributes:
 * `typeahead-editable` <i class="glyphicon glyphicon-eye-open"></i>
    _(Defaults: true)_ :
    Should it restrict model values to the ones selected from the popup only ?
+   
+* `typeahead-focus-first`
+   _(Defaults: true)_ :
+   Should the first match automatically be focused as you type?
```

Comment:
When I was looking for where to add the documentation for the new attribute, I noticed that they were almost in alphabetical order (except for focus-first) and assumed that they were supposed to be so I moved 'focus-first' to higher up in the list.  Looking again, I see I misplaced 'select-on-exact' anyway... so I've fixed that now.  I'm happy to order them differently if I'm incorrect.